template/src/pageComponents/mint/steps/MintCompleteStep/MintCompleteStep.tsx

```diff
@@ -0,0 +1,41 @@
+import { useCallback } from 'react';
+import clsx from 'clsx';
+import { MintSteps } from '../../ContractDemo';
+
+type MintCompleteStepProps = {
+  setMintStep: React.Dispatch<React.SetStateAction<MintSteps | null>>;
+  collectionName: string | null;
+};
+
+export default function MintCompleteStep({ setMintStep, collectionName }: MintCompleteStepProps) {
+  const handleMintAnother = useCallback(() => {
+    setMintStep(null);
+  }, [setMintStep]);
+
+  return (
+    <div
+      className={clsx(
+        'rounded-lg border border-boat-color-palette-line',
+        'mb-8 bg-boat-footer-dark-gray p-8',
+      )}
+    >
+      <h2 className="mb-5 w-full text-center text-2xl font-semibold text-white">
+        Congrats! You minted {collectionName}
+      </h2>
+
+      <div className="text-center text-6xl">🎉</div>
+
+      <div className="my-4 text-center text-sm text-gray-400">
+        It will take ~ 5 minutes to show up in your wallet
+      </div>
+
+      <button
+        type="button"
+        className="block w-full rounded-full bg-white py-4 text-center text-sm text-black"
+        onClick={handleMintAnother}
+      >
+        Mint another NFT
+      </button>
```

Comment:
I feel like we should standardize this button style/logic into `components/Button.tsx` so we can keep the same logic style everywhere.

By doing that, if someone wants to than replace it with their own Design System, is relativelly much faster.

Reply:
Good idea. I'll do this.

Reply:
Done :)